lighthouse-cli/test/fixtures/debugger.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<!--
+ * Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+-->
+<html>
+<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
+<body>
+  Take THAT Lighthouse.
+
+  <script>
+    setTimeout(() => {
+      debugger;
```

Comment:
```suggestion
      debugger; debugger;
```

😈 

(in words: I think this fix will only resolve the simplest of cases... we should elevate to v8 team?)

Reply:
Do tell, I'm not sure why :)

I made it more nefarious on your behalf

Reply:
I figured it would hit the next break point when resumed, but I am now realizing you are listening to the pause event (I thought it was just resuming once). so nvm